packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -417,12 +415,12 @@ class _CupertinoTabScaffoldState extends State<CupertinoTabScaffold> {
               // our own listener to update the [_controller.currentIndex] on top of a possibly user
               // provided callback.
               child: widget.tabBar.copyWith(
-                currentIndex: _controller.index,
+                currentIndex: _controller!.index,
                 onTap: (int newIndex) {
-                  _controller.index = newIndex;
+                  _controller!.index = newIndex;
                   // Chain the user's original callback.
                   if (widget.tabBar.onTap != null)
-                    widget.tabBar.onTap(newIndex);
+                    widget.tabBar.onTap!(newIndex);
```

Comment:
Could this be on one line?
```dart
widget.tabBar.onTap?.call(newIndex);
```

Reply:
Definitively. It's a matter of style. Done.